priv/data/community/blogs.exs

```diff
@@ -0,0 +1,82 @@
+[
+  %{
+    name: "Dashbit",
+    link: "https://dashbit.co/blog/",
+    icon: "/images/community/blogs/dashbit.png",
+    logo: nil,
+    about:
+      "Maintainers and contributors to many Elixir projects, such as Ecto, Broadway, LiveView, and Elixir itself."
+  },
+  %{
+    name: "OTP Team",
+    link: "https://blog.erlang.org/",
+    icon: "/images/community/blogs/otp.png",
+    logo: nil,
+    about:
+      "The Erlang/OTP team at Ericsson, the implementors and maintainers of Erlang/OTP."
+  },
+  %{
+    name: "Elixir Lang",
+    link: "https://elixir-lang.org/blog/",
+    icon: "/images/community/blogs/elixir.png",
+    logo: nil,
+    about:
+      "Official blog of the Elixir programming language."
+  },
+  %{
+    name: "Gleam News",
+    link: "https://gleam.run/news/",
+    icon: "/images/community/languages/gleam.svg",
+    logo: nil,
+    about:
+      "Latest official news on the Gleam programming language."
+  },
+  %{
+    name: "LFE blog",
+    link: "https://blog.lfe.io/",
+    icon: "/images/community/blogs/lfe.png",
+    logo: nil,
+    about:
+      "Official blog of the LFE programming language."
+  },
+  %{
+    name: "Planet Erlang",
+    link: "http://planeterlang.com/",
+    icon: nil,
+    logo: nil,
+    about:
+      "Planet Erlang is a meta blog that collects posts from the blogs of various Erlang developers and contributors."
+  },
+  %{
+    name: "Elixir Radar",
+    link: "https://elixir-radar.com/",
+    icon: nil,
+    logo: nil,
+    about:
+      "Keep updated with what's happening in the Elixir community."
```

Comment:
Hi, Hugo from Elixir Radar here! 🙂 

Can you change the about value to:

```
Elixir Radar is a curated content weekly Elixir newsletter and a job board for Elixir developers.
```

"

Reply:
Thank you very much Hugo, both for the text and for Elixir Radar! :) I've updated the copy.